src/components/calcite-block/calcite-block.tsx

```diff
@@ -127,7 +150,8 @@ export class CalciteBlock {
           <h3 class={CSS.heading}>{heading}</h3>
           {summary ? <div class={CSS.summary}>{summary}</div> : null}
         </div>
-        <slot name={CONTROL_SLOT_NAME} />
+        {loading || disabled ? null : <slot name={CONTROL_SLOT_NAME} />}
```

Comment:
I think this can be simplified to:

```tsx
const content = loading ?
      <calcite-loader inline is-active></calcite-loader> :
      !disabled ? <slot name={CONTROL_SLOT_NAME}/> :
        null;
```